### ISSUE TITLE
Unintuitive behavior with keybind example

### ISSUE THREAD

**Issue by qoh** [shanecelis/bevy_minibuffer_inspector#2]
[The README gives](https://github.com/shanecelis/bevy_minibuffer_inspector#inspect_world) this as one of the first examples:
```rust
...
    app
        .add_plugins(MinibufferPlugins)
        .add_acts((
            BasicActs::default(),
            inspector::WorldActs::default()
                .configure("inspect_world", |mut act| {
                    act.bind(keyseq! { I W });
                }),
        ));
}
```

I tried this since I'd like a keybind for it, but this leads to quite inconvenient behavior, although it eventually makes sense why:

Pressing **I W** anywhere in my game does trigger the act, but this doesn't actually visibly do anything. You won't see the inspector until the next time you open the minibuffer, since [it's only visible when the minibuffer also is](https://github.com/shanecelis/bevy_minibuffer_inspector/blob/5cb6f3f90bbe830b06cfc60ab972a1f23fcd56ee/src/world_inspector.rs#L81).
This would be better if either the act had `ActFlags::ShowMinibuffer` or if [the bind only listened when the minibuffer was already open](https://github.com/shanecelis/bevy_minibuffer/issues/8).

When you do have it open (visibly), closing it is a bit fiddly. As a user the first thing I tried is Esc, and after that didn't do anything I pressed **\`** to close the minibuffer. This does hide the inspector, but it doesn't actually close it; the next time you open the minibuffer it'll still be there. If you press the original **I W** bind again, this will close the inspector, but not the minibuffer, so the full sequence would actually be **I W \`**, or **\` I W**.
This would be better if the inspector closed when the minibuffer is closed, and/or if the minibuffer closed when the inspector is closed.

**Comment by shanecelis** [comment]
Right you are, it should have a `ShowMinibuffer` flag set.

The inspector can hold a lot of state and context, so I wouldn't want to close it just because the minibuffer was hidden. Would you want to decouple minibuffer visibility from the inspector visibility entirely?
